XMonad/Actions/TreeSelect.hs

```diff
@@ -258,6 +259,7 @@ defaultNavigation = M.fromList
 -- Using nice alternating blue nodes
 tsDefaultConfig :: TSConfig a
 tsDefaultConfig = def
+{-# DEPRECATED tsDefaultConfig "Use def (from Data.Default, and re-exported by XMonad.Actions.TreeSelect) instead." #-}
```

Comment:
Isn't the reexport from `XMonad` enough?

Reply:
Every module that has a `Data.Default` instance exports this again.  I wasn't even aware that `XMonad` re-exports it (it's quite well hidden, via `XMonad.Config`s re-export of `Default(...)`).  

I'm fine with removing the export, but if we are going for consistency then I think we'd have to remove it from all other modules as well.

Reply:
Oh, I grepped for "re-exported" and missed the other re-exports then.
Let's keep consistency then.

Reply:
The grep for `re-exported` doesn't return anything because all of these deprecations were removed in 5140f5b5d06790e055eb7fb0cf3eccc4997aa736.  If you go to that commit you'll see that all of these modules had a deprecations message just like that and all of them exported `def` as well